nuxeo-full-ps-template.yaml

```diff
@@ -0,0 +1,1992 @@
+apiVersion: v1
+kind: Template
+message: |-
+  The following backing services for Nuxeo have been created in your project:
+
+    * a MongoDB instance
+    * an Elasticsearch simple node
+    * a dev deployment
+    * a uat deployment
+    * a CI/CD pipeline build
+    * a Release pipeline build
+
+  Please add a Github webhook to https://openshift.dev.nuxeo.io/oapi/v1/namespaces/<<NAMESPACE>>/buildconfigs/${APPLICATION_NAME}-pipeline/webhooks/${GITHUB_WEBHOOK_SECRET}/github
```

Comment:
it has the dev cluster address hardcoded ; I don't think it can be parametrized, document it instead?